solutions_toolkit/uipath_block_scripts/utils.py

```diff
@@ -13,4 +13,6 @@ def files_from_directory(src_dir, regex="*.*"):
 
 
 def move_file(filepath, outdir):
-    shutil.move(filepath, outdir)
```

Comment:
Why are we doing this?

Reply:
To enable overwriting of duplicates in the UPLODED_DIR after moving to destination.

Reply:
I see, I need to test this out on my own system.